packages/angular/src/builders/ng-packagr-lite/ng-packagr-adjustments/write-bundles.ts

```diff
@@ -14,104 +14,11 @@ import {
   provideTransform,
   TransformProvider,
 } from 'ng-packagr/lib/graph/transform.di';
-import {
-  EntryPointNode,
-  isEntryPoint,
-  isEntryPointInProgress,
-} from 'ng-packagr/lib/ng-package/nodes';
-import { NgEntryPoint } from 'ng-packagr/lib/ng-package/entry-point/entry-point';
-import { rollupBundleFile } from 'ng-packagr/lib/flatten/rollup';
-import * as log from 'ng-packagr/lib/utils/log';
-import { DependencyList } from 'ng-packagr/lib/flatten/external-module-id-strategy';
-import { BuildGraph } from 'ng-packagr/lib/graph/build-graph';
-import { unique } from 'ng-packagr/lib/utils/array';
 
 export const nxWriteBundlesTransform: Transform = transformFromPromise(
-  async (graph) => {
-    const entryPoint = graph.find(isEntryPointInProgress()) as EntryPointNode;
-    const {
-      destinationFiles,
-      entryPoint: ngEntryPoint,
-      tsConfig,
-    } = entryPoint.data;
-    const cache = entryPoint.cache;
-
-    // Add UMD module IDs for dependencies
-    const dependencyUmdIds = entryPoint
-      .filter(isEntryPoint)
-      .map((ep) => ep.data.entryPoint)
-      .reduce((prev, ep: NgEntryPoint) => {
-        prev[ep.moduleId] = ep.umdId;
-
-        return prev;
-      }, {});
-
-    const { fesm2015, esm2015 } = destinationFiles;
-
-    const opts = {
-      sourceRoot: tsConfig.options.sourceRoot,
-      amd: { id: ngEntryPoint.amdId },
-      umdModuleIds: {
-        ...ngEntryPoint.umdModuleIds,
-        ...dependencyUmdIds,
-      },
-      entry: esm2015,
-      dependencyList: getDependencyListForGraph(graph),
-    };
-
-    log.info('Bundling to FESM2015');
-    // @ts-ignore
-    cache.rollupFESMCache = await rollupBundleFile({
-      ...opts,
-      moduleName: ngEntryPoint.moduleId,
-      format: 'es',
-      dest: fesm2015,
-      // @ts-ignore
-      cache: cache.rollupFESMCache,
-    });
-  }
+  (graph) => graph
```

Comment:
guess this file still needs to be present because the NgPackagr DI requires it, right?

Reply:
Yep, it's used by the [entryPointTransformFactory](https://github.com/ng-packagr/ng-packagr/blob/v11/src/lib/ng-package/entry-point/entry-point.transform.ts#L35) within ng-packager. I guess I can provide my custom `entryPointTransformFactory` that does not try to write bundles, it'd look cleaner than this noop function. What do you think?

Reply:
yeah probably. In general I think at some point we might just drop the ng-packagr dependency because I feel we're customizing now everything basically :)